processor/src/advice/inputs.rs

```diff
@@ -10,54 +10,54 @@ use super::{utils::IntoBytes, BTreeMap, Felt, InputError, MerkleSet, Vec};
 ///
 /// There are three types of advice inputs:
 ///
-/// 1. Single advice tape which can contain any number of elements.
-/// 2. Multiple advice tapes that can be appended to the main tape, and are mapped by 32 bytes keys.
+/// 1. Single advice stack which can contain any number of elements.
+/// 2. Key-mapped stacks set that can be pushed onto the operand stack.
```

Comment:
This should say "Key-mapped element lists which can be pushed onto the advice stack."

Reply:
Fixed on #785